setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
     install_requires=(
         'Django>=1.5',
         'easy-thumbnails>=1.0',
-        'django-mptt>=0.6',
+        'django-mptt>=0.6,<0.8',
```

Comment:
The problem is all the other applications which declare dependency on filer, which must pin the mptt version whenever they depend on filer, which seems rather unconfortable